torchvision/transforms/v2/functional/_utils.py

```diff
@@ -37,7 +37,22 @@ def decorator(kernel):
     return decorator
 
 
+def _name_to_dispatcher(name):
+    import torchvision.transforms.v2.functional  # noqa
+
+    try:
+        return next(
```

Comment:
Not the most efficient, we could store the mapping somewhere. I don't think we care anyway since this is just executed during registration, not when calling the dispatcher.

Reply:
Not sure I understand the iteration here. Can't we just do

```python
try:
    return getattr(torchvision.transforms.v2.functional, name)
except AttributeError:
    raise ValueError(...) from None
```

Reply:
yes much better!